patches/youch@4.1.0-beta.10.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/build/chunk-4XB2BYKC.js b/build/chunk-4XB2BYKC.js
+index 6e6e490115bc4b107b53834ceac3d7d19cdbe3fe..37373d734d034caf45e28c1b165e5c176f901137 100644
+--- a/build/chunk-4XB2BYKC.js
++++ b/build/chunk-4XB2BYKC.js
+@@ -34,10 +34,10 @@ var ErrorStackSource = class extends BaseComponent {
+    */
+   async toHTML(props) {
+     const frame = props.frame;
+-    if (frame.type === "native" || !frame.source || !frame.fileName) {
++    if (!frame.source || !frame.fileName) {
+       return "";
+     }
+-    const language = LANGS_MAP[extname(frame.fileName)] ?? "plain";
++    const language = LANGS_MAP[extname(frame.fileName)] ?? (frame.type === "native" ? "js" : "plain");
+     const highlightMarginTop = `${frame.source.findIndex((chunk) => {
+       return chunk.lineNumber === frame.lineNumber;
+     }) * 24}px`;
+diff --git a/build/chunk-JAN2TFI2.js b/build/chunk-JAN2TFI2.js
+index a6321161988abe906e7745bf9d540670dee5b289..bcbc6e02801718867bb7249e47e1af04e155df7e 100644
+--- a/build/chunk-JAN2TFI2.js
++++ b/build/chunk-JAN2TFI2.js
+@@ -10,6 +10,7 @@ import {
+ } from "./chunk-4YEN7HVQ.js";
+
+ // src/templates/error_stack/main.ts
++import { existsSync } from "node:fs"
+ import { dump, themes } from "@poppinss/dumper/html";
+ import { dump as dumpCli } from "@poppinss/dumper/console";
+ var CHEVIRON = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" width="24" height="24" stroke-width="2">
+@@ -50,7 +51,7 @@ var ErrorStack = class extends BaseComponent {
+    */
+   #getEditorLink(ide, frame) {
+     const editorURL = EDITORS[ide] || ide;
+-    if (!editorURL || frame.type === "native") {
++    if (!editorURL || frame.type === "native" || !existsSync(frame.fileName)) {
```

Comment:
Not sure if this is a good patch upstream as it assumes youch is running on node environment only. But I will raise it up when I submit a PR to youch.